FlowCrypt/Core/Core.swift

```diff
@@ -62,8 +56,13 @@ class Core: KeyDecrypter, KeyParser, CoreComposeMessageType {
 
     // MARK: - Setup
     func setupWebView() {
-        guard let jsFileSrc = getCoreJsFile() else { return }
+        let userController = WKUserContentController()
```

Comment:
Here you updated only initial webView initialization, but we also need to use it when `WKErrorDomain` was caught.

Reply:
Well, when `WKErrorDomain` was caught this `setupWebView` function would be called and webview should be re-initialized.

```swift
if error._domain == "WKErrorDomain" {
    // Core js code injected using evaluateJavaScript result is removed when app is in background for long time
    // Need to setup again. https://github.com/FlowCrypt/flowcrypt-ios/issues/2013
    setupWebView()
    return try await call(endpoint, params: params, data: data)
}
```

Reply:
Ah, I missed it 👍
I'll install new build and check if bug is gone in the evening